requirements.txt

```diff
@@ -121,7 +121,7 @@ kombu==4.6.8
     # via
     #   -r requirements.in
     #   celery
-lxml==4.6.3
+lxml==4.6.1
```

Comment:
Seems unrelated?

Reply:
This was a regression to test CodeQL as this version of `lxml` contains a couple of known vulnerabilities, but it didn't seem to pick them up.